### DISCUSSION TITLE
Add Stockholder Approval Date to StockPlan Object

### DISCUSSION THREAD

**Discussion by JSv4** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#150] [Suggestions] [answered]
**Per Issue #56, importing comments / suggestions to our static Google Doc into GitHub for discussion**

As pointed out by @ashawber, for ISOs, we need stockholder-approved stock plans. He proposes we add a field to [StockPlan Objects](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/main/schema/objects/StockPlan.schema.json) for `stockholder_approval_date`.

**Comment by JSv4** [comment, marked as answer]
From our discussion on 3/29/22 - it would be very helpful to be explicit why this is included. Ok as an optional field.

**Comment by ashawber** [comment]
Under US federal income tax regs, to receive ISO treatment for stock options, the underlying stock plans must be approved by stockholders within 12 months of the plan being adopted by the board. If this requirement isn’t met, then any stock options granted under the plan will be treated as NSOs. That’s why it’s necessary to track the stockholder approval date.

On Mar 29, 2022, at 9:53 AM, JSIV ***@***.***> wrote:

﻿

From our discussion on 3/29/22 - it would be very helpful to be explicit why this is included. Ok as an optional field.

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/discussions/150*discussioncomment-2463428__;Iw!!K_R5ZAeIjLw!WvzEfDUcPgFR65iT4xBOCuXVU0uwTR_2NK1ySQ0ijz5CNpiYZqeT86oJ5RfMEbbnJ_g$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AX5MIGXK5XTLZA5ZGFEAP23VCMYR3ANCNFSM5R4HMPOA__;!!K_R5ZAeIjLw!WvzEfDUcPgFR65iT4xBOCuXVU0uwTR_2NK1ySQ0ijz5CNpiYZqeT86oJ5RfMjR0mZPU$>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

________________________________

NOTICE: This communication may contain privileged or other confidential information. If you have received it in error, please advise the sender by reply email and immediately delete the message and any attachments without copying or disclosing the contents. Thank you.
